src/htcondor_es/history.py

```diff
@@ -21,7 +21,7 @@
 from htcondor_es.convert_to_json import unique_doc_id
 
 
-def process_schedd(starttime, last_completion, checkpoint_queue, schedd_ad, args, metadata=None):
+def process_schedd(starttime, last_completion, checkpoint_queue, schedd_ad, args, metadata=None,kwargs={}):
```

Comment:
Why do you allow any keyword arguments here, but only accept `aff_mgr` when you pass them to `convert_to_json`? You could just either add a single one, or allow any and then check in `convert_to_json` whether they are present. (Same below for the `process_queues`.)